openassessment/assessment/serializers/base.py

```diff
@@ -48,18 +49,12 @@ class Meta:
 
 class RubricSerializer(serializers.ModelSerializer):
     """Serializer for :class:`Rubric`."""
-    criteria = CriterionSerializer(required=True, many=True)
+    criteria = CriterionSerializer(required=False, many=True)
```

Comment:
Why the required attribute is False here?

Reply:
@Itezaz-ul-Hassan Because we don't want to make it mandatory to add a rubric criteria to an ORA block